File: Exemplos/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/TERMICA/impressao_qr_code.dart
import 'package:intent_digital_hub/IntentDigitalHubService/TERMICA/termica_commad.dart';

class ImpressaoQRCode extends TermicaCommand {
  final String _dados;
  final int _tamanho;
  final int _nivelCorrecao;

  ImpressaoQRCode(this._dados, this._tamanho, this._nivelCorrecao)
      : super('ImpressaoQRCode');

  @override
  functionParameters() {
    return '"dados"'
        ':'
        '"$_dados"'
        ','
        '"tamanho"'
        ':'
        '$_tamanho'
        ','
        '"nivelCorrecao"'
        ':'
        '$_nivelCorrecao';
  }
}

File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/ImpressaoQRCode.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class ImpressaoQRCode(
    private val dados: String,
    private val tamanho: Int,
    private val nivelCorrecao: Int
) : TermicaCommand("ImpressaoQRCode") {
    override fun functionParameters(): String {
        return "\"dados\"" + ":" + "\"" + dados + "\"" + "," +
                "\"tamanho\"" + ":" + tamanho + "," +
                "\"nivelCorrecao\"" + ":" + nivelCorrecao
    }
}